components/Dialogs/src/private/MDCAlertActionManager.m

```diff
@@ -89,6 +89,7 @@ - (MDCButton *)makeButtonForAction:(MDCAlertAction *)action
                             target:(id)target
                           selector:(SEL)selector {
   MDCButton *button = [[MDCButton alloc] initWithFrame:CGRectZero];
+  button.uppercaseTitle = NO;
```

Comment:
I don't think we should make this change as the default. M1 styled dialogs still should use all-uppercase, and I believe it's still Material's "default" look for titles.  If we wanted to change Button default values, we should probably do it in Buttons (globally) rather than only in Dialogs.

Reply:
The M1 dialogs would still be all caps, because `-uppercaseString` would be called in the factory methods [here](https://github.com/material-components/material-components-ios/pull/5795/files#diff-7f12d761bb6d1e1c91da59ada3276844R51), but yeah, sounds like you all have a solution, so I'll close this